parserator_web/static/js/index.js

```diff
@@ -1,2 +1,41 @@
 /* TODO: Flesh this out to connect the form to the API and render results
    in the #address-results div. */
+document.addEventListener("DOMContentLoaded", function () {
```

Comment:
let's split this function into a few smaller functions that 

* make the API call
* writes the table if successful
* displays the error if not successful.